python/vpc/vpc-flow-logs/README.md

```diff
@@ -0,0 +1,62 @@
+# AWS VPC with Flow Logs with Python CDK!
```

Comment:
Please replace this automatically-generated README with a README that describes your example.